coding-concepts.js

```diff
@@ -8,39 +8,39 @@
 // --------------------1) What will this log?
 
 var cohort = "Alpha 2022"
-// console.log(cohort.split(""))
+console.log(cohort.split(""))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: ['A', 'l', 'p', 'h', 'a', ' ', '2', '0', '2', '2']
+// b) Verify and explain: Output -> ['A', 'l', 'p', 'h', 'a', ' ', '2', '0', '2', '2']. My answer was correct. By using . split(""), it will separate the string to make an array. By putting the "" inside the parentheses, each character in the string will be in quotes in the array.
 
 
 // --------------------2) What will this log?
 
 const greeter = (name) => {
   `Hello, ${name}!`
 }
-// console.log(greeter("LEARN Student"))
+console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 'Hello, LEARN Student!'
+// b) Verify and explain: Output -> undefined. This is because every function requires a return which is what was missing here. By having a return statment, the function returns a value. 
```

Comment:
This one was a tricky one to catch!  Recognizing why it was not the expected outcome is super important though, so great work in recognizing this!